hudi-sync/hudi-hive-sync/src/main/java/org/apache/hudi/hive/HiveSyncConfigHolder.java

```diff
@@ -72,7 +72,7 @@ public class HiveSyncConfigHolder {
       .withDocumentation("Hive metastore url");
   public static final ConfigProperty<String> HIVE_AUTO_CREATE_DATABASE = ConfigProperty
       .key("hoodie.datasource.hive_sync.auto_create_database")
-      .defaultValue("true")
+      .defaultValue("false")
       .markAdvanced()
```

Comment:
Okay with it, just need to make sure the effect of existing use cases.

Reply:
It depends on how the tables/databases are organized and how automations are setup on top of deltastreamer. In our case, we rely on this to create the DB if not present instead of having a separate job (or manual ways) to create database. I think it would be safe to leave it like this instead of changing defaults.

Reply:
Yeah, the legacy jobs compatibility is an issue, that's why the PR is suspended.